docs/send-data/kubernetes/troubleshoot-collection.md

```diff
@@ -209,6 +209,101 @@ otellogs:
 
 For more information, see [Setting different resources on different nodes for logs collector](/docs/send-data/kubernetes/best-practices/#setting-different-resources-on-different-nodes-for-logs-collector).
 
+### Check logs body
+
+You can print logs on stdout of logs collector and logs metadata, and validate if they are correct. It may happen that logs are ingested, but with different metadata than you expect them.
+
+In order to print them on stdout, two steps are required:
+
+1. Disable ingesting logs from logs related pods. This is required to prevent logs ingest spike. Add the following configuration to `user-values.yaml`:
```

Comment:
```suggestion
1. Disable log ingestion from logs-related pods. This is required to prevent logs ingest spike. Add the following configuration to `user-values.yaml`:
```